.build/parent-pom-template.xml

```diff
@@ -805,7 +805,20 @@
       <dependency>
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcpkix-jdk18on</artifactId>
-        <version>1.76</version>
+        <version>1.78</version>
```

Comment:
I see version 1.78.1 now available, also showing more downloads than 1.78. Looking at the release notes it's probably not something that might affect us, but I thought I would let you know about that version.